docs/getting-started/running-release.md

```diff
@@ -21,17 +27,12 @@ Use [Deploying Release Version of the bpfman-operator](#deploying-release-versio
 below for deploying released version of `bpfman` in Kubernetes and then use the
 links above for further information on how to test and interact with `bpfman`. 
 
-> **NOTE:**
-> The latest release, v0.3.1, was before the rename of `bpfd` to `bpfman`.
-> So the commands below still refer to bpfd.
-
 ## Local Host
 
-To run `bpfd` in the foreground using `sudo`, download the release binary tar
-files and unpack them.
+To run `bpfman` in the foreground, download the release binary tar files and unpack them.
 
 ```console
-export BPFMAN_REL=0.3.1
+export BPFMAN_REL=0.4.0
 mkdir -p $HOME/src/bpfman-${BPFMAN_REL}/; cd $HOME/src/bpfman-${BPFMAN_REL}/
 wget https://github.com/bpfman/bpfman/releases/download/v${BPFMAN_REL}/bpfd-linux-x86_64.tar.gz
 tar -xzvf bpfd-linux-x86_64.tar.gz; rm bpfd-linux-x86_64.tar.gz
```

Comment:
This (`bpfctl`) reference is stale

Reply:
Good catch. Fixed to get `bpfman` and `bpfman-rpc`. However, I can't really test this until you build the release packages, so guessing on names based on the last release of bpfd.